website/src/components/Sidebar/index.jsx

```diff
@@ -0,0 +1,100 @@
+// @flow
+import React, { Fragment } from 'react';
+import Link from 'gatsby-link';
+import styled, { css } from 'styled-components';
+import { colors as akColors } from '@atlaskit/theme';
+import { colors } from '../../constants';
+import type { docsPage, sitePage } from '../types';
+import { getTitleFromExamplePath } from '../../utils';
+
+const Sidebar = styled.div`
```

Comment:
I would ask that these be based on the grid, but given that i plan on moving the sidebar out we can leave it like this for now